packages/synapse-react-client/src/synapse-queries/entity/useExportToCavatica.ts

```diff
@@ -21,7 +24,7 @@ export function useExportToCavatica(
     try {
       // add drs_uri to select
       const selectColumnsList = selectColumns?.map(col => col.name).join(',')
-      const sql = `SELECT CONCAT('drs://repo-prod.prod.sagebase.org/syn', id, '.', currentVersion) AS drs_uri, name as file_name, ${selectColumnsList} FROM ${originalSql.slice(
+      const sql = `SELECT CONCAT('drs://repo-prod.prod.sagebase.org/syn', ${fileIdColumnName}, '.', ${fileVersionColumnName}) AS drs_uri, ${fileNameColumnName} as file_name, ${selectColumnsList} FROM ${originalSql.slice(
```

Comment:
might be worth checking this SQL in useExportToCavatica.test.ts under the default case and custom parameters case

`expect(mockGetDownloadFromTableRequest.mock.calls[0][0]).equals('SELECT ...")`

Reply:
Good idea